tests/50federation/35room-invite.pl

```diff
@@ -250,11 +250,12 @@ sub do_v1_invite_request
    my ( $room, $first_home_server, $outbound_client, $invitation ) = @_;
 
    my $room_id = $room->room_id;
+   my $event_id = $room->id_for_event( $invitation );
```

Comment:
this is a slightly unrelated change, but the warning messages were confusing me.